nncf/experimental/common/tensor_statistics/statistics.py

```diff
@@ -237,7 +299,7 @@ class WCTensorStatistic(TensorStatistic):
     SHAPE_STAT = "shape_values"
 
     mean_values: List[Tensor]
-    shape_values: List[Tuple[int, ...]]
+    shape_values: List[Tuple[Tensor]]
```

Comment:
?
`        self.shape_values = [tuple(shape.tolist()) for shape in loaded_data[self.SHAPE_STAT]]`

Reply:
done